integration_tests/models/cross_db_utils/test_listagg.sql

```diff
@@ -0,0 +1,69 @@
+with data as (
+
+    select * from {{ ref('data_listagg') }}
+
+),
+
+data_output as (
+
+    select * from {{ ref('data_listagg_output') }}
+
+),
+
+calculate as (
+
+    select
+        group_col,
+        {{ dbt_utils.listagg('string_text', "','", "order by order_col") }} as actual,
+        1 as version
```

Comment:
I like that all of these are in the one model, but I'm wondering if we can move away from magic numbers, and instead show what the goal of the version is. Something like `comma_ordered`, `comma_ordered_limited`, `comma_unordered`, `distinct_comma`, `no_params`...

Also, just noticed that all of these use `,` as their delimiter, which is also the default value, so again it'd be good to test it with something different to make sure that it's doing what we ask it to instead of being correct by accident. 

It'd be particularly worthwhile to use a multi-character/whitespace-using delimiter for a bit of novelty, e.g. `, ` or something weird like `_|_`

Reply:
Love these ideas! Thanks for all of the feedback :)

Reply:
Note " , " won't work as a delimiter if you have whitespace in your `measure` column. This is in-line with the note that "if there are instances of `delimiter_text` within your measure, you cannot include a `limit_num`".